java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -1546,6 +1546,25 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_stringReplace(JNIEnv *env
   CATCH_STD(env, 0);
 }
 
+JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_stringReplaceMulti(JNIEnv *env, jclass,
+                                                                          jlong column_view,
+                                                                          jlong targets, jlong repls) {
```

Comment:
All these parameters are column view.
```suggestion
                                                                          jlong input_cv,
                                                                          jlong targets_cv, jlong repls_cv) {
```